forui/test/src/widgets/line_calendar_golden_test.dart

```diff
@@ -0,0 +1,110 @@
+@Tags(['golden'])
+library;
+
+import 'package:flutter/widgets.dart';
+
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:forui/forui.dart';
+import 'package:forui/src/widgets/line_calendar/line_calendar.dart';
+import '../test_scaffold.dart';
+
+void main() {
```

Comment:
The `TestScaffold` shouldn't need additional padding and centering surrounding it.